packages/plugin-network-breadcrumbs/network-breadcrumbs.js

```diff
@@ -190,6 +204,35 @@ module.exports = (_ignoredUrls = [], win = window) => {
     }
   }
 
+  const getByteLength = (body) => {
+    // body could be any of the types listed here: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/send#parameters
+    // ReadableStreams cannot be read safely and it's difficult to get an accurate byte length for FormData and Document inputs
+    if ((body === null || typeof body === 'undefined') ||
+      (win.ReadableStream && body instanceof win.ReadableStream) ||
+      (win.FormData && body instanceof win.FormData) ||
+      (win.Document && body instanceof win.Document)) return undefined
```

Comment:
Should we test this is an instance of the class rather than an instance of itself?
```suggestion
      (win.ReadableStream && body instanceof ReadableStream) ||
      (win.FormData && body instanceof FormData) ||
      (win.Document && body instanceof Document)) return undefined
```

Reply:
This wouldn't pass the linting checks as those types are not globally defined (but they are exposed on `window`) - discussed here: https://bugsnag.slack.com/archives/G03JXULFW06/p1674737016675659